plugin/src/main/java/com/denizenscript/denizen/objects/properties/entity/EntityJumpStrength.java

```diff
@@ -7,14 +7,17 @@
 import com.denizenscript.denizencore.objects.properties.Property;
 import com.denizenscript.denizencore.tags.Attribute;
 import org.bukkit.entity.EntityType;
+import org.bukkit.entity.Donkey;
 import org.bukkit.entity.Horse;
+import org.bukkit.entity.Mule;

```

Comment:
Every change in this class is doing it entirely wrong

Reply:
Handled instead via https://github.com/DenizenScript/Denizen/commit/4610e88b1866f2fd00b6bc4e9d7c23b9fe9e7302 so just remove your changes to this class